Automattic/wp-calypso
# Invite list: add ability to resend

This PR follows #21538 and adds a "Resend Invite" option to pending invites.

Ref: p3Ex-32O-p2

![35302094-fa5278d0-0052-11e8-9f65-7a3bef826075](https://user-images.githubusercontent.com/942359/35458390-22f259aa-02aa-11e8-9850-d44f9e59f3eb.png)

**To Test:**
- Visit your invites list.
- On accepted invites, no resend option should be available.
- On pending invites, a "Resend Invite" button should be available.
- Clicking the button should resend the invite. A `calypso:invites-actions` debug event is available with the siteId and inviteId.
- Clicking the same invite resend again while the request is still being made (the button will be busy) should not duplicate the request.
- Clicking multiple different invites should be allowed.
- After the invite is resent, either a success or error notice should be displayed.
- The "Resend Invite" button should not be shown on mobile screens - that will be handled on a separate details page in the future.
